src/cases/components/case-hearings-list/case-hearings-list.component.ts

```diff
@@ -26,6 +27,7 @@ export class CaseHearingsListComponent implements OnInit {
   public hasUpdateAction: boolean = false;
   public hasDeleteAction: boolean = false;
   public hasReadOnlyAction: boolean = false;
+  public sub: Subscription;
```

Comment:
is this sub still needed?

Reply:
I have removed this code which is not required.